### PR TITLE
✨Prensa Ibérica amp-ad support

### PR DESCRIPTION
**Pull request by sbonetclement**
* Adding support for prensa ibérica amp-ad

### CODE REVIEW

**Comment by sbonetclement** [comment]
Hi @powerivq, 
could you review this pull request please?
Thank you

File: 3p/integration.js [hunk 1]
```diff
@@ -199,6 +199,7 @@ import {openadstream} from '#ads/vendors/openadstream';
 import {openx} from '#ads/vendors/openx';
 import {opinary} from '#ads/vendors/opinary';
 import {outbrain} from '#ads/vendors/outbrain';
+import {piberica} from '#ads/vendors/piberica';
```
Comment (powerivq) [inline 3p/integration.js:202]: Please remove changes to this file. This is not necessary for new networks.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: 3p/integration.js (hunk 1 above)
Comment (sbonetclement) [reply 3p/integration.js]: I just removed them. Thanks

File: 3p/integration.js [hunk 2]
```diff
@@ -491,6 +491,7 @@ register('openadstream', openadstream);
 register('openx', openx);
 register('opinary', opinary);
 register('outbrain', outbrain);
+register('piberica', piberica);
```
Comment (powerivq) [inline 3p/integration.js:494]: This needs to be removed as well.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: 3p/integration.js (hunk 2 above)
Comment (sbonetclement) [reply 3p/integration.js]: Removed.

**Review by powerivq — APPROVED** [review]

**Pull request merged**